components/call-to-action/call-to-action.module.scss

```diff
@@ -31,18 +31,39 @@
     p {
       margin-bottom: 3rem;
     }
+  }
+
+  &__link-container {
+    $bp-links-move-inline: 37.5rem;
+
+    display: flex;
+    flex-direction: column;
+
+    @media (min-width: $bp-links-move-inline) {
+      flex-direction: row;
+    }
 
     a {
-      width: 15rem;
+      width: 100%;
```

Comment:
I think we may also want `box-sizing: border-box` on the links, otherwise they extend past their parent's container.

![Screen Shot 2022-09-07 at 8 21 30 AM](https://user-images.githubusercontent.com/82047862/188904628-40bc2392-617f-41f8-b7e2-214e5987379e.png)

Reply:
@dustin-jw I can add that!

Reply:
There's also something on the page that causes horizontal overflow on small screens (<350px), but I couldn't find what it was, and it wasn't relevant to your changes, so 🤷

Reply:
Hmm this is interesting. But unrelated, you're right. 
![Screen Shot 2022-09-07 at 11 10 21 AM](https://user-images.githubusercontent.com/7475148/188913898-5f269524-4a32-4bce-af05-6c6d6de38d35.png)